scripting/anko.md

```diff
@@ -203,7 +203,7 @@ The following functions are only available in scripts implementing the `Main` fu
 * `readEntry() entry, error` returns the next entry and an error (if any). It will return an error when no entries remain.
 * `writeEntry(ent) error` writes out the given entry down the pipeline, returning an error if any.
 * `cloneEntry(ent) entry` returns a copy of the specified entry.
-* `newEntry() entry` creates an entirely new entry, with the timestamp set to the current time.
+* `newEntry() entry` creates an entirely new entry, with the timestamp set to the end of the current query.
```

Comment:
not true, the prototype for the function is: `func(ts time.Time, data []byte)`

Reply:
different functions in anko module vs. scheduled searches, see #96